### DISCUSSION TITLE
[求助]绳网UI炸了？

### DISCUSSION THREAD

**Discussion by lcp1234** [share121/inter-knot#649] [其他]
下午1:00进去的还好好的，现在变成这个样子了
![Screenshot_20240810_131826_mark via](https://github.com/user-attachments/assets/2f3e71b4-424f-4e29-8be9-d705980a11df)
![Screenshot_20240810_131742_mark via](https://github.com/user-attachments/assets/0c0fa246-17c3-486e-b1a0-cb554c86a9fa)

**Comment by Hongxuli2006** [comment]
我还好啊，绳网没啥问题啊。你要不重新启动或者换个网试试看

**Comment by Hongxuli2006** [comment]
你们手机是怎么搞得，我只能PC

**Reply by lcp1234** [reply]
via可以装油猴脚本。

**Reply by c328422** [reply]
請問是哪個腳本，現在我也有安裝但無論顯示，是無法連線

**Comment by hzjwork** [comment]
我这也是，好像是把电脑的UI做的更适配更大了一些，然后手机就遭殃了

**Comment by Yizakl** [comment]
一个样，ui突然变得很大

**Comment by Qijoten** [comment]
同意，怕不是出了点问题，我的也是这样，后面好了点

**Comment by Anonymous755** [comment]
我也是

**Comment by lllliufeng** [comment]
升级了吧

**Comment by ZhuYuan-148** [comment]
这位绳匠，可开启浏览器的电脑模式来解决

**Comment by shengwanguser002** [comment]
点右上角三个杠换成电脑模式

**Comment by Hongxuli2006** [comment]
我去邮箱突然间来了这么多消息吓我一跳。还以为是爆仓了

**Reply by lcp1234** [reply]
？

**Comment by lllliufeng** [comment]
这版是不是把标题的特殊字符扬了（

**Comment by cheese502** [comment]
炸了之后莫名编辑不了帖子了，在线等挺急的

**Comment by hjejdjdj** [comment]
改电脑模式就行了

**Comment by capacitor1** [comment]
> Reply::炸了之后莫名编辑不了帖子了，在线等挺急的

可以直接用Github浏览并编辑的 绳网实际上读取的就是Discussion的内容 如果网页出了bug仍然可以直接用GitHub参与讨论

**Comment by kunga555** [comment]
不知道，反正手机没什么问题

**Reply by Yizakl** [reply]
？过去这么久的贴，你到底翻了多久啊

**Reply by lcp1234** [reply]
idk,我只知道现在绳网网页端已经没了，需要访问，只能Github discussion或者手机端APP。

**Comment by c328422** [comment]
更新的狀況是現在只能用舊的討論串無法創建新的，UP關閉分類了，說是有更重要的事情要做，我dc問到的是這樣，就看誰要把自己的討論串貢獻出來作為新繩網討論串囉（APP跟git帳號討論串聯結）好像有釋放項目的實現方式
